src/documentation/search/search.ts

```diff
@@ -127,11 +127,18 @@ const exploreSearchableContent = (toolkit: Toolkit): SearchableContent[] => {
     t.contents?.forEach((e) => {
       const contentString = blocksToText(e.content);
       const detailContentString = blocksToText(e.detailContent);
+      const alternativesLabel = "\n\n" + defaultString(e.alternativesLabel);
```

Comment:
Maybe we should serialize these where they appear in the content?

We could make the blocks -> text conversion support serializers for non-"block" _type blocks. Not sure whether this is worth it but maybe let's discuss later.

Reply:
Oh no, ignore me, they're a separate field.